ShopGo/HomeScreen/HomeViewController.swift

```diff
@@ -8,11 +8,26 @@
 import UIKit
 
 class HomeViewController: UIViewController {
-
+    
+    let cartfloatingButton = FloatingCart().cartfloatingButton
+    
     override func viewDidLoad() {
         super.viewDidLoad()
+        view.addSubview(cartfloatingButton)
+        cartfloatingButton.addTarget(self, action: #selector(cartFloatingButtonTapped), for: .touchUpInside)
+        NSLayoutConstraint.activate([
```

Comment:
make a separate function for this and call it in viewDidLoad

Reply:
Fixed